publication-event-handlers/build.gradle

```diff
@@ -3,6 +3,7 @@ dependencies {
     implementation project(":storage-model")
     implementation project(":expansion")
     implementation project(":publication-commons")
+    implementation project(":publication-rest")
```

Comment:
necessary for getting the CreatePublicationRequest object that is now in the rest-package